### ISSUE TITLE
triple asteriks mark down for bold and italic doesnt work

### ISSUE THREAD

**Issue by FireCubeStudios** [BreeceW/Oxidized#45]
when i send a message with triple asteriks it doesnt show bold and italic but only bold surrounded by asteriks.

version: 1.7.3

windows version: Windows 10 Home Insider Preview 10.0.21296

System: Surface Pro 6 (bios 235.3261.768) running Intel ( R) Core (TM) i5-8250U CPU 

bug: 
![image](https://user-images.githubusercontent.com/40955345/106433048-f042f700-646f-11eb-998e-dd6cf83f2385.png)

expected behaviour:
![image](https://user-images.githubusercontent.com/40955345/106433124-06e94e00-6470-11eb-9dfc-59fe3507653e.png)

Steps to reproduce issue:
1) open oxidized
2) go to valid area to send message
3) in the text box enter text surrounded by 3 asteriks on both sides and send message. (Example: ***e***)

How to fix:
Add support for markdown

Reasons to fix: triple asteriks allows expression of emotion bypassing capital letters and sets aggressive tone

**Comment by BreeceW** [comment]
Yes, this will require some reworking of the message parser, which is planned eventually.
